docs/details/commands/pmm-admin.md

```diff
@@ -323,6 +323,59 @@ When you remove a service, collected data remains on PMM Server for the specifie
         - `push`: agent will push metrics.
         - `pull`: server scrapes metrics from agent.
 
+## Advanced Options
+
+### Enable all collectors
+
+PMM starts the MongoDB exporter by default by using these these parameters:
+
+|Parameter|Description|
+|-----|-----|
+|`--collector.diagnosticdata`|Enables `getDiagnosticData` collector|
+|`--collector.replicasetstatus`|Enables `replSetGetStatus` collector|
+|`--discovering-mode`|Auto discovers all databases and collections|
+|`--compatible-mode`|Enables MongoDB exporter v1 compatible metric names|
+|`--mongodb.global-conn-pool`|Uses a single connection to the DB instead of connecting in each scrape|
+|`--collector.collstats-limit=200`|Enables collStats and indexStats only if there are less than n collections in total|
```

Comment:
@denisok , @percona-csalguero  did we keep 200 as default? how this will affect users?

Reply:
that is example I think, not default. But yes, Puneet tested it after this doc. @percona-csalguero what is the default now?

Reply:
it should be the default flags for 2.26.0

Reply:
```
AddMongoDBC.Flag("max-collections-limit",
		"Disable collstats & indexstats if there are more than <n> collections. 0: No limit, -1: Let PMM automatically set this value.").
		Default("-1").Int32Var(&AddMongoDB.CollectionsLimit)
```

currently in code